pkg/loki/config_wrapper.go

```diff
@@ -509,6 +509,18 @@ func applyStorageConfig(cfg, defaults *ConfigWrapper) error {
 		}
 	}
 
+	if !reflect.DeepEqual(cfg.Common.Storage.AlibabaCloud, defaults.StorageConfig.AlibabaStorageConfig) {
+		configsFound++
+
+		applyConfig = func(r *ConfigWrapper) {
+			r.Ruler.StoreConfig.Type = "alibabacloud"
+			r.Ruler.StoreConfig.AlibabaCloud = r.Common.Storage.AlibabaCloud
+			r.StorageConfig.AlibabaStorageConfig = r.Common.Storage.AlibabaCloud
+			r.CompactorConfig.SharedStoreType = config.StorageTypeAlibabaCloud
+			r.StorageConfig.Hedging = r.Common.Storage.Hedging
+		}
+	}
+
```

Comment:
FYI changes in Loki should go in separate PR. We try to keep PRs affecting one project at a time.

Reply:
Can you help me how to remove commit from this PR?

Reply:
You can add an additional commit on this PR with deleting this change. It does not matter, we will squash all commits before merging.